uint8 id_modulo
uint8 submodo
bool activo